docs/src/pages/careers/lead-designer.md

```diff
@@ -69,10 +70,9 @@ As a **team leader**, you will:
 
 We're looking for someone with an expert eye for both UI and UX design. Details matter to you.
 
-You're a leader who can support and guide a small but growing design team.
+You're a manager that can support and guide the team.
```

Comment:
Since we say "You have either managed a team before or are feeling ready to start now" down the line, I think that we go one step back. I don't think that it's so much a manager (I'm afraid it could attract people that can't do IC work very well) that we need at this stage, but 1. someone to lead (coordinate the direction for the design team) 2. and someone to **support** the design team. It's the beginning of what a manager would do, but not full-time.

```suggestion
You're a leader who can support and guide a small but growing design team.
```